lib/bloc/account/account_model.dart

```diff
@@ -404,6 +408,7 @@ class AccountModel {
 
     if (outgoing && amount > maxAllowedToPay) {
       if (reserveAmount > 0) {
+        return null; // TODO: remove this workaround when we have a solution to decimals round at our library
```

Comment:
@ademar111190 I think we intended to disable the validation only in the case the user marked "Use all funds" in the UI. This validation now is disabled for all cases.

Reply:
🤦 true, sorry I forget it